models/ast/node_evaluation_dto.go

```diff
@@ -10,10 +13,22 @@ type NodeEvaluationDto struct {
 }
 
 func AdaptNodeEvaluationDto(evaluation NodeEvaluation) NodeEvaluationDto {
-	return NodeEvaluationDto{
+	nodeEvaluationDto := NodeEvaluationDto{
 		ReturnValue:   evaluation.ReturnValue,
 		Errors:        pure_utils.Map(evaluation.Errors, AdaptEvaluationErrorDto),
 		Children:      pure_utils.Map(evaluation.Children, AdaptNodeEvaluationDto),
 		NamedChildren: pure_utils.MapValues(evaluation.NamedChildren, AdaptNodeEvaluationDto),
 	}
+
+	if isReturnValueOmitted(evaluation) {
+		nodeEvaluationDto.ReturnValue = omittedReturnValue
+	}
+
+	return nodeEvaluationDto
+}
+
+const omittedReturnValue = "__omitted_return_value"
+
+func isReturnValueOmitted(evaluation NodeEvaluation) bool {
+	return evaluation.Function == FUNC_CUSTOM_LIST_ACCESS
```

Comment:
I'm open to discussion on this. What I need, is an "explicit" way to discover omitted result in the frontend (and I prefer a rule that is not based on business logic like the one implemented in `isReturnValueOmitted`).